contributing/styling.md

```diff
@@ -0,0 +1,85 @@
+[&larr; Table of Contents](../CONTRIBUTING.md)
+
+# Styling Guidelines
+🎨 How to CSS in FDS
+
+## Styling components
+
+Prefer CSS custom properties over hard coded values. The [style dictionary](https://cbinsights.github.io/form-design-system/fds-dictionary/) in fds lists available custom properties.
+
+```diff
+.foo {
+-  background-color: #ff6633;
++  background-color: var(--color-orange);
+}
+```
+
+Prefer existing utility classes over writing new CSS
+
+```diff
+-.fooComponent {
+-  padding-top: var(--space-default);
+-  color: var(--color-secondary);
+-  display: inline-block;
+-}
++<div className="padding--top color--secondary display--inlineBlock">
+```
+
+## Managing specificity
+**Aim to keep selectors to a single class**. Avoid ancestry and combinators.
+
+CSS classes in FDS follow an _additive class pattern_. A base class should define default
+styles of an element, and additional classes can be added to modify style properties.
+
+```
+/* "base" includes all default styles. Variant classes only change what is necessary */
+<Component className="base base--variant base--anotherVariant" />
+```
+
+**Avoid binary class switching**. Use an additive class pattern instead.
+
+```diff
+-className={isSelected ? 'menu-item-selected' : 'menu-item-normal'}
++className=cx('menu-item', { 'menu-item--selected': isSelected })
```

Comment:
Is it worth calling out the actual classnames (or classcat) package? Essentially it's not easily possible to do this additive approach without these packages.

Reply:
i opted for the familiar library here - we just have to remember to search `./contributing` for usage when we switch to classcat

Reply:
right. So what I mean here is like... let's assume classcat doesn't exist. I wonder if developers even know what `cx` is.

Reply:
good point - going with plain JS for this example.